main.js

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
there's lots of inconsistent indentation in this file too, which makes me think our eslint configuration isn't working properly. let's investigate that today